tests/contracts/test_concise_contract.py

```diff
@@ -0,0 +1,48 @@
+
+import sys
+
+import pytest
+
+from web3.contract import (
+    ConciseContract,
+    ConciseMethod,
+)
+
+if sys.version_info >= (3, 3):
```

Comment:
When do we get to stop supporting py2? I'm so ready.

Reply:
I'm going to formally announce it at devcon after which **all-the-things** will get warnings that support is being dropped.

I suppose we could drop some `PendingDeprecationWarning` warnings in sooner.  I'm game.